Package@swift-6.swift

```diff
@@ -0,0 +1,927 @@
+  static var nioEmbedded: Self { .product(name: "NIOEmbedded", package: "swift-nio") }
+  static var nioExtras: Self { .product(name: "NIOExtras", package: "swift-nio-extras") }
+  static var nioFoundationCompat: Self { .product(name: "NIOFoundationCompat", package: "swift-nio") }
+  static var nioHTTP1: Self { .product(name: "NIOHTTP1", package: "swift-nio") }
+  static var nioHTTP2: Self { .product(name: "NIOHTTP2", package: "swift-nio-http2") }
+  static var nioPosix: Self { .product(name: "NIOPosix", package: "swift-nio") }
+  static var nioSSL: Self { .product(name: "NIOSSL", package: "swift-nio-ssl") }
+  static var nioTLS: Self { .product(name: "NIOTLS", package: "swift-nio") }
+  static var nioTransportServices: Self {
+    .product(
+      name: "NIOTransportServices",
+      package: "swift-nio-transport-services"
+    )
+  }
+  static var nioTestUtils: Self { .product(name: "NIOTestUtils", package: "swift-nio") }
+  static var nioFileSystem: Self { .product(name: "_NIOFileSystem", package: "swift-nio") }
+  static var logging: Self { .product(name: "Logging", package: "swift-log") }
+  static var protobuf: Self { .product(name: "SwiftProtobuf", package: "swift-protobuf") }
+  static var protobufPluginLibrary: Self {
+    .product(
+      name: "SwiftProtobufPluginLibrary",
+      package: "swift-protobuf"
+    )
+  }
+  static var dequeModule: Self { .product(name: "DequeModule", package: "swift-collections") }
+  static var atomics: Self { .product(name: "Atomics", package: "swift-atomics") }
+  static var tracing: Self { .product(name: "Tracing", package: "swift-distributed-tracing") }
+
+  static var grpcCore: Self { .target(name: "GRPCCore") }
+  static var grpcInProcessTransport: Self { .target(name: "GRPCInProcessTransport") }
+  static var grpcInterceptors: Self { .target(name: "GRPCInterceptors") }
+  static var grpcHTTP2Core: Self { .target(name: "GRPCHTTP2Core") }
+  static var grpcHTTP2TransportNIOPosix: Self { .target(name: "GRPCHTTP2TransportNIOPosix") }
+  static var grpcHTTP2TransportNIOTransportServices: Self { .target(name: "GRPCHTTP2TransportNIOTransportServices") }
+}
+
+// MARK: - Targets
+
+extension Target {
+  static var grpc: Target {
+    .target(
+      name: grpcTargetName,
+      dependencies: [
+        .cgrpcZlib,
+        .nio,
+        .nioCore,
+        .nioPosix,
+        .nioEmbedded,
+        .nioFoundationCompat,
+        .nioTLS,
+        .nioTransportServices,
+        .nioHTTP1,
+        .nioHTTP2,
+        .nioExtras,
+        .logging,
+        .protobuf,
+        .dequeModule,
+      ].appending(
+        .nioSSL, if: includeNIOSSL
+      ),
+      path: "Sources/GRPC",
+      swiftSettings: [.swiftLanguageVersion(.v5)]
+    )
+  }
+
+  static var grpcCore: Target {
+    .target(
+      name: "GRPCCore",
+      dependencies: [
+        .dequeModule,
+        .atomics
+      ],
+      path: "Sources/GRPCCore",
+      swiftSettings: [.swiftLanguageVersion(.v5)]
```

Comment:
Do we want to adopt Swift 6 language version for the v2 packages? Or not yet?

Reply:
I'm going to make the changes in a separate PR to keep the diff small(er). This way we can do it module by module (or at least do the larger ones separately)